collections/DOrville/MS_DOrville_153.xml

```diff
@@ -37,45 +40,132 @@
                      <idno type="SCN">17031</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>Boethius, <title>De consolatione philosophiae</title>, with glosses and commentary; Germany (?), c. 1438</head>
                <msContents>
+                  <textLang mainLang="la">Latin</textLang>
+                  <msItem>
+                     <locus>(fols. ii recto – verso)</locus>
+                     <title type="desc">Commentary on an unidentified text</title>
+                     <incipit>Hic autor p<ex>onit</ex> d<ex>iffinitionem</ex> aliquorum nominum et dicit quod sertum est pilleus virginalis floribus herbis compositus</incipit>
+                     <incipit><locus>(fol. ii verso)</locus> Hic autor ponit diffinitionem dicens quod aula dicitur domus comitis</incipit>
+                     <incipit>Hic autor ponit diffinitionem dicens quod alea dicitur quidam asser quadratus</incipit>                     
+                     <note>A reject leaf, the commentary text unfinished and the lemmata not added, with later pen trials in the margins. </note>
+                     <note>The lower part of fol. ii verso with various quotations: the first is: ‘Est vitrum saphirus saphyrus splendida gemma’</note>
+                  </msItem>
+                  <msItem>
+                     <locus>(fol. iii recto)</locus>
+                     <note>Blank except for: ‘Item Paulus apostolus | Qui in uno offendit est omnium reus’ (cf. James 2:10)</note></msItem>
+                  <msItem>
+                     <locus>(fol. iii verso)</locus>
+                     <note>Blank except for:</note>
+                     <note>  ‘Item septem sunt que per oculis habere debemus <lb/> Primum est brevitas vite nostre <lb/> Secundum est certitudo mortis. <gap reason="editorial"/> Septem perdicio eterne glorie.’</note>
```

Comment:
Is `<space>` the element to use here rather than `<gap>`, if I am understanding correctly that this meant to reflect the page?